Sources/Widgets/Widgets3D/LineWidget/Constants.js

```diff
@@ -13,7 +13,8 @@ export const HandleRepresentationType = {
   SPHERE: 'sphere',
```

Comment:
I do not see the difference between HandleBehavior and widgetState.nbHandles. Don't you have some duplicated information ?

Please review the usefulness of HandleBehavior